citations-tool/tool/src/webapp/js/jquery.googlebooks.thumbnails.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * JQuery Plugin to get thumbnail images from Google Books
+ *
+ *  Given an img (or list of img's), which is representing a thumbnail for a media type,
+ *  that has a class of 'citationThumbnail' and an id containing its ISBN,
+ *  this plugin populates the src of that img with a small thumbnail from the Google Books API.
+ */
+(function ($) {
+
+    var GOOGLE_BOOKS_SEARCH_API = 'https://www.googleapis.com/books/v1/volumes?';
+
+    $.fn.setGoogleBooksThumbnailURL = function(element) {
+        if (element.id!=''){
+            $.getJSON(GOOGLE_BOOKS_SEARCH_API + 'q=isbn:' + element.id, function(data){
+                var imgURL;
+                try {
+                    imgURL = data.items[0].volumeInfo.imageLinks.smallThumbnail;
+                    element.src = imgURL;
+                }
+                catch(err) {}
+            });
+        }
+    };
+
+    $(document).ready( function() {
```

Comment:
You want to have this outside the plugin so you can take any jQuery selector and apply the google books plugin to it.